wherescape/connectors/hubspot/python_hubspot_check_missing_columns.py

```diff
@@ -111,3 +119,35 @@ def compare_load_and_ds(wherescape_instance, table_rows):
         % wherescape_instance.load_full_name,
         new_rows_in_load_table,
     )
+
+
+def create_metadata(wherescape_instance):
```

Comment:
I assume this works. did not test it. does it work for creating both a load_table and a ds_table???